config/locales/en.yml

```diff
@@ -349,18 +349,22 @@ en:
   show_live_form:
     contact_details: Your form’s contact details
     declaration: Declaration
+    declaration_description: Your form's declaration is shown to people when they have answered all the questions, just before they submit the form.
     edit_link: Edit this form
-    form_content: Form content
     privacy_policy_link: Privacy policy link
-    questions:
-      one: 1 question
-      other: "%{count} questions"
-      zero: no quesions
+    questions: Questions
+    questions_link:
+      one: View your form's 1 question
+      other: View your form's %{count} questions
+      zero: no questions
```

Comment:
🤔 should there be zero questions? also if there are zero questions what should we show?